lib/services/usagereporter.go

```diff
@@ -68,6 +69,8 @@ const (
 	usageReporterRetryAttempts = 5
 )
 
+var defaultTeleportRoles = []string{"access", "auditor", "editor"}
```

Comment:
Move this to `constants.go`, under `PresetAuditorRoleName` (and use those consts).

Might wanna add `admin` to the list, too.

Reply:
Ouu, [done](https://github.com/gravitational/teleport/pull/21123/commits/d0703124270a858fd173958ebbf95683ee7d36e6)